plugins/woocommerce/tests/e2e-pw/tests/merchant/create-post.spec.js

```diff
@@ -27,7 +27,7 @@ test.describe( 'Can create a new post', () => {
 						force: true,
 					},
 				} );
-				expect( response.ok() ).toBeTruthy();
+//				expect( response.ok() ).toBeTruthy();
```

Comment:
Good call commenting this out. I think we should remove the assertion, and clean up the response variable above it. I created a separate task for it in https://github.com/woocommerce/woocommerce/issues/38195.